node-operators/block-producers/operational-tasks/upgrade-your-node.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Upgrade your Node
```

Comment:
This should be something more descriptive for SEO purposes. This is the title that is shown on search engines, so it should be something more like "Update Your Block Producing Node" or something more explicit - remember that `Tanssi Docs` are appended to the title, so we don't need to explicitly say "Tanssi" in the title